### PR TITLE
Pedal form frirst

### PR DESCRIPTION
**Pull request by JonasDaJones**
Last suggestions implemented. nothing new.

### CODE REVIEW

File: components/form-category-input/category-input.styled.js
Comment (JessicaLoers) [inline components/form-category-input/category-input.styled.js:10]: Although it is not part of the PR, I would like to mention that this is kind of funky 😬.
I suggest splitting the child styles into another component (something like this not tested example below) but it's up to you.

```

 const StyledCategoryItem = styled.div`
    color: whitesmoke;
    font-weight: bold;
    position: relative;
`;

const StyledCategoryView = styled.div`
  width: 100%;
  height: 3em;
  border: 2px solid #2490b5;

  ${StyledCategoryItem} {
    color: whitesmoke;
    font-weight: bold;
    position: relative;
  }
`;
```

File: components/form-category-input/index.js
Comment (JessicaLoers) [inline components/form-category-input/index.js:30]: Clean Code rules - please remove this comment before the merge 🧹

```suggestion
         
```

File: components/form-tag-input/tag-input.styled.js
Comment (JessicaLoers) [inline components/form-tag-input/tag-input.styled.js:8]: Please delete the commented out code  🧹

File: components/new-pedal-form/index.js
Comment (JessicaLoers) [inline components/new-pedal-form/index.js:64]: I was a bit confused that your new entries are not displayed - and then was glad that you have this console.log. Still, we want to avoid merging log messages into our main.

**Review by JessicaLoers — APPROVED** [review]
**Hi @JonasDaJones **, well done.  👍

I have a few comments that you are welcome to look at. 
I talked to Julia that you have been working on this feature for a while and I think at the beginning of the weekend you have earned an approval. 🚀

**Review by JessicaLoers — COMMENTED** [review]
**Hi @JonasDaJones **, well done.  👍

I have a few comments that you are welcome to look at. 
I talked to Julia that you have been working on this feature for a while and I think at the beginning of the weekend you have earned an approval. 🚀

**Comment by JonasDaJones** [comment]
Thank you for your patience and really important suggestions. That means a lot to me. And yes, I was thinking about getting all useState() calls into one but then forgot it. I take it as my homework to learn being more patient and accurate before sending PR (or esp. merge). Have a nice weekend!

File: components/form-category-input/category-input.styled.js
Comment (JonasDaJones) [reply components/form-category-input/category-input.styled.js]: as in the form the child does not need to be styled, i deleted this. btw. to avoid problematic prop-drilling, i took the categories out from their own component and implemented them to new-pedal-form.

File: components/form-category-input/index.js
Comment (JonasDaJones) [reply components/form-category-input/index.js]: done

File: components/new-pedal-form/index.js
Comment (JonasDaJones) [reply components/new-pedal-form/index.js:26]: for a further idea, (direct editing of just one part of the object by doubleclick in detail-view without having to open the whole form) i thought it would be handy if they have their own state...

File: components/new-pedal-form/index.js
Comment (JonasDaJones) [reply components/new-pedal-form/index.js]: completely replaced by: const updatedPedals = [...pedals, newPedal];
    savePedals(updatedPedals);
    handleReset();

File: components/new-pedal-form/index.js
Comment (JonasDaJones) [reply components/new-pedal-form/index.js]: done

File: components/new-pedal-form/index.js
Comment (JonasDaJones) [reply components/new-pedal-form/index.js]: now new pedal is shown in pedal gallery on home

File: components/pedal-details-main/index.js
Comment (julialoeschel) [inline components/pedal-details-main/index.js:29]: same here - i think this `p` should be an `li`

File: pages/index.js
Comment (julialoeschel) [inline pages/index.js:9]: we try to avoid drilling the setter Function. check teh state session again to handle the state. [here](https://github.com/neuefische/muc-web-23-3/blob/main/sessions/react-state-2/react-state-2.md#sharing-state-between-components)

File: pages/new-pedal.js
Comment (julialoeschel) [inline pages/new-pedal.js:10]: ah look. you did this already. 
For better namingconvention, this could be 
```suggestion
      <NewPedalForm pedals={pedals} onHandlePedalSubmit={handlePedalSubmit} />
```

**Review by julialoeschel — COMMENTED** [review]
really good job. 
a little big, try to make the next one smaller :) 

I left some hints for you. 
🐜 
keep on going!

File: components/pedal-details-main/index.js
Comment (JonasDaJones) [reply components/pedal-details-main/index.js:29]: here i agree

**Pull request merged**
